src/fragments/lib/in-app-messaging/respond-interaction-events/respond-interaction-events.mdx

```diff
@@ -63,9 +63,10 @@ listener.remove(); // Remember to remove the listener when it is no longer neede
 If you are using the Amplify In-App Messaging UI, interaction events notifications are already wired up for you. However, if you are implementing your own UI, it is highly recommended to notify listeners of interaction events through your UI code so that the library can take further actions prescribed by the installed provider (for example, automatically recording corresponding Analytics events).
 
 <Callout informational>
-  If you are using TypeScript for your development, you can import and use the
-  InAppMessageInteractionEvent enum instead of hard-coding the interaction event
-  strings below.
+
+  If you are using TypeScript for your development, you can import and use the `InAppMessageInteractionEvent` enum
```

Comment:
can we provide an example of the import as well, please? Just to make it super clear